### ISSUE TITLE
(tlg0062-Harmon) updating the Harmon Greek and adding his English

### ISSUE THREAD

**Issue by gregorycrane** [PerseusDL/canonical-greekLit#1469]
# Basic Content
This. pull request includes:

1. Corrections to the Harmon Greek editions of Lucian (TLG 1-52). Many were made manually as I worked through the texts. The fact that we had entered this edition for 1st1k Greek also and that helped give me a second text for comparison. About 2/3 of the corrections are accent fixes (usually where grave accents appeared before enclitics) and places where the original OCR missed a raised dot.

There are so many fixes that it will not be possible to review them all. I recommend spot checking. I removed the \<corr/> tags around the c. 1,800 automatic corrections. I did not find any issues with these corrections and I decided it was best simply to publish a clean text.

2. Notes were extracted from the 1st1k Greek version and folded into the Perseus edition. Once this pull request is complete, we can retire the 1st1k version.

3. I have added the Harmon translations (and notes). This OCR could benefit from systematic correction but is good enough to get started. It contains notes also. I have validated the fact that each Greek text and English translation has the same number of books/sections but I know that there are occasional places where the section break needs tweaking. Often it it is hard to tell from the Loeb where the break should be.

# Updates to the format

1. Notes now have language attributes.

2. We have the xml:base attribute added.

3. The \_\_cts\_\_.xml files now have \<ti:description/> attributes that follow are newer guidelines.

4. I have removed the birth and death dates for Harmon but think it important to include the full name (Austin Morris Harmon). That information greatly helps me (at least) when I try to get more information on the editor/translator. My goal would be to add that information over time to every TEI header but I would start adding this to new files.

## For the English

1. \<resp/> statement now follows the standard model (following the header to tlg0550.tlg001 as an example).

2. We now have \<editor role="translator">

3. Since these translations are based directly on perseus-grc2, the version perseus-eng2 replaces 1st1k-eng1. 

## For the Greek

1. An addition to the \<revisionDesc> element identifies the addition of notes and correction of errors as the major content changes.

**Comment by lcerrato** [comment]
Was this fixed? I see several PRs.
